### PR TITLE
New MOD Suit UI

### PR DESCRIPTION
**Pull request by MTandi**
SbrdSE19m6

<details><summary>More pictures</summary>
<p>

E8CBXwEyyu

xhljZ3b2V0

</p>
</details> 

## About The Pull Request

Refactored MOD UI.

## Why It's Good For The Game

MOD UI is now more compact and easier to use.

## Changelog

:cl:
refactor: Refactored MOD Suit UI
qol: Added options to hide detailed vitals and DNA data to the MOD status module
qol: Made MOD Ion Jetpack Module have stabilizers enabled by default
/:cl:

### CODE REVIEW

**Comment by atlasle** [comment]
very very clean, this is great

File: tgui/packages/tgui/interfaces/MODsuit.tsx [hunk 1]
```diff
@@ -0,0 +1,723 @@
...
+                average: [0.2, 0.5],
+                bad: [0.5, Infinity],
+              }}>
+              <AnimatedNumber value={active ? loss_tox : 0} />
+            </ProgressBar>
+          </LabeledList.Item>
+        )}
+        {background_radiation_level !== undefined && (
+          <LabeledList.Item label="Radiation">
+            {active ? (
```
Comment (jlsnow301) [inline tgui/packages/tgui/interfaces/MODsuit.tsx:525]: This block gets a little hectic

File: tgui/packages/tgui/interfaces/MODsuit.tsx [hunk 2]
```diff
@@ -0,0 +1,723 @@
...
+    </Window>
+  );
+};
+
+export const MODsuitContent = (props, context) => {
+  const { act, data } = useBackend<MODsuitData>(context);
+  const { interface_break } = data.suit_status;
+  return (
+    <Box>
+      {(!!interface_break && <LockedInterface />) || (
```
Comment (jlsnow301) [inline tgui/packages/tgui/interfaces/MODsuit.tsx:117]: Use a ternary?

File: tgui/packages/tgui/interfaces/MODsuit.tsx [hunk 3]
```diff
@@ -0,0 +1,723 @@
...
+                </Table.Cell>
+                <Table.Cell textAlign="center">{module.use_power}</Table.Cell>
+                <Table.Cell textAlign="center">
+                  {module.module_complexity}
+                </Table.Cell>
+              </Table.Row>
+            );
+          })}
+        </Table>
+      )) || <NoticeBox>No Modules Detected</NoticeBox>}
```
Comment (jlsnow301) [inline tgui/packages/tgui/interfaces/MODsuit.tsx:720]: Not terrible but I feel like this makes it easy to lose your place. Can you early return this?

**Review by jlsnow301 — COMMENTED** [review]
Solid. And you put it in typescript, blessed move. My only concerns are nits, mostly in your usage of nested ternaries

**Comment by Pickle-Coding** [comment]
Charge should be in joules instead of watts.

**Pull request merged**